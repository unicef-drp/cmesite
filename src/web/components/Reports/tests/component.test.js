import React from 'react';
import { shallow } from 'enzyme';
import Component from '../component';

describe('Reports | component', () => {
  describe('render', () => {
    it('should match snapshot', () => {
      const root = <Component />;

      const wrapper = shallow(root);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
