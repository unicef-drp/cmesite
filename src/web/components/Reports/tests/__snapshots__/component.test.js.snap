// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reports | component render should match snapshot 1`] = `
<Reports
  classes={
    Object {
      "action": "Reports-action-11",
      "area": "Reports-area-6",
      "btnType": "Reports-btnType-14",
      "card": "Reports-card-5",
      "container": "Reports-container-4",
      "content": "Reports-content-8",
      "leftIcon": "Reports-leftIcon-13",
      "media": "Reports-media-10",
      "reports": "Reports-reports-1",
      "secondaryCard": "Reports-secondaryCard-7",
      "secondaryContent": "Reports-secondaryContent-9",
      "secondaryWrapper": "Reports-secondaryWrapper-3",
      "typo": "Reports-typo-12",
      "wrapper": "Reports-wrapper-2",
    }
  }
  reports={Array []}
/>
`;
